fusor-ember-cli/app/templates/content-source-upstream.hbs

```diff
@@ -1,16 +0,0 @@
-<h1>No subscriptiosn for upstream</h1>
-
-<p>
-  There are no subscriptions for upstream.
-</p>
-
-<p>
-  Do we need to define the source(s) of the content?
-</p>
-
-
-
-<div style='text-align:right'>
-  <button {{action 'showModal' 'cancel-modal'}} class='btn btn-default'>Cancel</button>
-  {{#link-to 'review.installation' class='btn btn-primary' disabled=disableTabInstallation}}Next{{/link-to}}
-</div>
```

Comment:
@knowncitizen, yes, this is old code when I thought to support an upstream project with no subscriptions. I removed the router and controller and guess I forgot to remove this template.